Nits2508 (#15)
mirjak/draft-dhc-dhcpv4-over-dhcpv6-ra: draft-ietf-dhc-dhcpv4-over-dhcpv6-ra.md

   {{RFC2131}} {{RFC2132}}, and DHCPv6 {{draft-ietf-dhc-rfc8415bis}}.

* Lightweight DHCPv6 Relay Agent (or LDRA):
   This is an extension of the original DHCPv6 Relay Agent specification, to allow
   layer-2-only devices to perform a Relay Agent function {{RFC6221}}.

* DHCPv4 over DHCPv6 Relay Agent (or 4o6RA):
   Refers to a Relay Agent that implements the 4o6

>    functionality specified in this document.

Reviewer: ```suggestion
   transport as specified in this document.
```
This aligns with the title of RFC7341

that apply to the DHCP client in {{Section 5 of RFC7341}} are also applied to the 4o6RA.

As the 4o6RA takes the role of the client in respect to {{RFC7341}},

> it is responsible for determining a suitable interface on which to be a

Reviewer: ```suggestion
it is responsible for determining a suitable interface where it acts as a
```

DHCPv6 client, and it is responsible for locating a suitable DHCPv6

> server or relay agent.
>    It is RECOMMENDED that the 4o6RA requests the DHCP 4o6
>    Server Address option from the server by sending the option code in
>    an Option Request option using an INFORMATION-REQUEST  as
>    described in {{draft-ietf-dhc-rfc8415bis}}.

Reviewer: ```suggestion
server or relay agent and obtain the necessary IPv6 configuration..
As specified in {{RFC7341}}, the 4o6RA, acting as 4o6 client, therefore has to request
the DHCP 4o6 Server Address option from the server by sending the
Option Request option as described in {{draft-ietf-dhc-rfc8415bis}}
before it can use the 4o6 transport.
```
It's not good practice to specify something normatively here again, which is already specified normatively somewhere else. There let's just give a reference and restate what's required by roc7341 already.

# Example Use Case: Topology Discovery for IPv4-only Radio Unit in 3GPP RAN with Switched Fronthaul {#usecase}

In 3GPP mobile network architecture, the User Equipments (UE) are connected
via Radio Access Network (RAN). RAN is built up with Baseband Units (BB)
and Radio Units (RU). Radio Fronthaul Network (FH) connects RU and BB, each of
RU and BB is an IP host, they may support IPv4 only, IPv6 only or both
depending on the vendor and the model.
Each RU is unique as it is tied to a set of antennas, and each antenna
is serving a specific Cell and Sector.
Each RU is configured by the BB depending on the Cell and Sectors it serves.
However, that dependency is only specified by the cabling between RU and antennas.

> 

Reviewer: ```suggestion
```
